OracleTuxedo/salt/samples/bankapp/start.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+cd /u01/data/bankapp/
+
+APPDIR=$(pwd)
+export APPDIR
+
+UNAME1=$(uname -n)
+TUX_UID=$(id -u)
+TUX_GID=$(id -g)
+
+
+# clean up from any previous run
+tmshutdown -y &>/dev/null 
+rm -f TLOG GWTLOG tuxconfig saltconfig bankdl1 bankdl2 bankdl3 ULOG.*
+
+
+
+# modify and run the environment setup script 
+cp -p bankvar bankvar.new
+
+echo '
+#
+# For GWWS
+#
+GWTLOGDEVICE=${APPDIR}/GWTLOG
+export GWTLOGDEVICE
+#
+# Device for binary file that gives SALT all its information
+#
+SALTCONFIG=${APPDIR}/saltconfig
+export SALTCONFIG
+' >> bankvar.new
```

Comment:
This should probably be in HEREDOC format.

Reply:
I am creating a new file "bankvar.new" by reusing the contents of existing "bankvar" file and then appending the above lines (without variable expansion). Not sure how HEREDOC format could be used for the same purpose.